packages/cashier/src/Components/side-note.jsx

```diff
@@ -1,39 +1,82 @@
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import React from 'react';
 import { DesktopWrapper, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
 import 'Sass/side-note.scss';
 
-const SideNoteBullet = ({ children }) => (
+const SideNoteTitle = ({ title }) => (
+    <Text className='side-note__title' weight='bold' as='p'>
+        {title}
+    </Text>
+);
+
+const SideNoteText = ({ children }) => (
+    <Text className='side-note__text' size='xxs' as='p'>
+        {children}
+    </Text>
+);
+
+const SideNoteBullet = ({ children, component }) => (
     <div className='side-note__bullet-wrapper'>
         <div className='side-note__bullet' />
-        <span>{children}</span>
+        {component ? children : <SideNoteText>{children}</SideNoteText>}
     </div>
 );
 
-const SideNote = ({ notes, title, has_bullets }) => (
-    <div className='side-note'>
-        {title && (
-            <DesktopWrapper>
-                <Text className='side-note__text' weight='bold' as='p'>
-                    {title}
-                </Text>
-            </DesktopWrapper>
-        )}
-
-        {notes.map((note, i) =>
-            has_bullets ? (
-                <SideNoteBullet key={i}>{note}</SideNoteBullet>
-            ) : (
-                <Text key={i} className='side-note__text' size='xxs' as='p'>
-                    {note}
-                </Text>
-            )
-        )}
-    </div>
-);
+const SideNote = ({ side_notes, title, has_bullets = true }) => {
+    const side_note_title =
+        title ||
+        (side_notes?.length > 1 ? <Localize i18n_default_text='Notes' /> : <Localize i18n_default_text='Note' />);
+
+    const checkNote = (note, i) => {
+        let Component;
+        if (typeof note === 'string') {
+            if (has_bullets)
+                return (
+                    <SideNoteBullet key={i}>
+                        <Localize i18n_default_text={note} />
+                    </SideNoteBullet>
+                );
+            return <SideNoteText key={note.key || i}>{note.i18n_default_text}</SideNoteText>;
+        } else if (typeof note === 'object' && note.props.i18n_default_text) {
+            Component = { ...note };
+            if (has_bullets)
+                return (
+                    <SideNoteBullet key={i} component>
+                        <SideNoteText>{Component}</SideNoteText>
+                    </SideNoteBullet>
+                );
+            Component.key = i;
+            return Component;
+        }
+
+        Component = { ...note };
+        Component.key = i;
+        return Component;
+    };
+
+    return (
+        <div className={classNames('side-note', { 'side-note--mobile': isMobile })}>
+            {side_note_title && (
```

Comment:
Is there any time that `side_note_title` will be false? If not please remove this condition.

Reply:
No, It always has value. I removed the condition.